modules/test/playwright/tests/product-navigation-user-personal-bar-web/userPersonalBar.spec.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {userPersonalBarPagesTest} from '../../fixtures/userPersonalBarPagesTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	userPersonalBarPagesTest,
+	loginTest
+);
+
+test('LPD-15423 notification badge is not visible when show notification badge in personal menu is disabled', async ({
+	apiHelpers,
+	userPersonalBarPage,
+}) => {
+	await userPersonalBarPage.goToProcessBuilderConfigurationTab();
+	await userPersonalBarPage.enableSingleApproverWorkflowProduct();
+	await userPersonalBarPage.disableNotificationBadgeInPersonalMenu();
+
+	const catalog = await apiHelpers.headlessCommerceAdminCatalog.postCatalog(
+		'Test Catalog'
+	);
+
+	const product = await apiHelpers.headlessCommerceAdminCatalog.postProduct(
+		catalog.id,
+		'Test Product'
+	);
+
+	const response = await apiHelpers.headlessCommerceAdminCatalog.patchProduct(
+		product.productId
+	);
+
+	expect(response.ok()).toBe(true);
+
+	await userPersonalBarPage.goto();
```

Comment:
I think you need to go to the "home page". You can use directly the page variable
You need to get it from 
```
async ({
	apiHelpers,
	page,
	userPersonalBarPage,
}
```